### PR TITLE
Add new properties to AKS

### PR DESCRIPTION
**Pull request by rahalan**
# Change

Adding new properties

[ContainerService: ManagedClusters](https://github.com/Azure/ResourceModules/actions/workflows/ms.containerservice.managedclusters.yml)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

## Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (readme)
- [x] I did format my code

### CODE REVIEW

File: arm/Microsoft.ContainerService/managedClusters/deploy.bicep [hunk 1]
```diff
@@ -421,7 +529,13 @@ output resourceGroupName string = resourceGroup().name
 output name string = managedCluster.name
 
 @description('The control plane FQDN of the managed cluster')
-output controlPlaneFQDN string = (aksClusterEnablePrivateCluster ? managedCluster.properties.privateFQDN : managedCluster.properties.fqdn)
+output controlPlaneFQDN string = (enablePrivateCluster ? managedCluster.properties.privateFQDN : managedCluster.properties.fqdn)
```
Comment (MrMCake) [inline arm/Microsoft.ContainerService/managedClusters/deploy.bicep:532]:
```suggestion
output controlPlaneFQDN string = enablePrivateCluster ? managedCluster.properties.privateFQDN : managedCluster.properties.fqdn
```

File: arm/Microsoft.ContainerService/managedClusters/deploy.bicep [hunk 2]
```diff
@@ -421,7 +529,13 @@ output resourceGroupName string = resourceGroup().name
...
 
 @description('The principal ID of the system assigned identity.')
 output systemAssignedPrincipalId string = systemAssignedIdentity && contains(managedCluster.identity, 'principalId') ? managedCluster.identity.principalId : ''
+
+@description('The Object ID of the AKS identity.')
+output kubeletidentityObjectId string = contains(managedCluster.properties, 'identityProfile') ? contains(managedCluster.properties.identityProfile, 'kubeletidentity') ? contains(managedCluster.properties.identityProfile.kubeletidentity, 'objectId') ? managedCluster.properties.identityProfile.kubeletidentity.objectId : '' : '' : ''
```
Comment (MrMCake) [inline arm/Microsoft.ContainerService/managedClusters/deploy.bicep:538]: Is the last check really necessary? -> Can there be a kubelet identity without an object Id?

Comment (rahalan) [reply arm/Microsoft.ContainerService/managedClusters/deploy.bicep]: removed

File: arm/Microsoft.ContainerService/managedClusters/deploy.bicep [hunk 3]
```diff
@@ -421,7 +529,13 @@ output resourceGroupName string = resourceGroup().name
...
+
+@description('The Object ID of the OMS agent identity.')
+output omsagentIdentityObjectId string = contains(managedCluster.properties, 'addonProfiles') ? contains(managedCluster.properties.addonProfiles, 'omsagent') ? contains(managedCluster.properties.addonProfiles.omsagent, 'identity') ? contains(managedCluster.properties.addonProfiles.omsagent.identity, 'objectId') ? managedCluster.properties.addonProfiles.omsagent.identity.objectId : '' : '' : '' : ''
```
[the remark this answers is not in the document]
Comment (rahalan) [reply arm/Microsoft.ContainerService/managedClusters/deploy.bicep]: removed

**Review by MrMCake — APPROVED** [review]

**Pull request merged**